regolith/templates/beamplan.txt

```diff
@@ -0,0 +1,77 @@
+\documentclass[prl,aps,tighten,amsmath,amssymb,floatfix]{revtex4-1}
+
+\usepackage{graphicx}
+
+\newcommand{\midrule}{\hline}
+\newcommand{\bottomrule}{\hline\hline}
+
+\begin{document}
+
+\title{The experiment plan for the beamtime {{bt}}}
+\author{Songsheng Tao}
```

Comment:
this needs a rethink.  I don't think we want hard-coded names here

Reply:
Maybe we don't really need names here. It is just an internal file.